ocaml/idl/datamodel_host.ml

```diff
@@ -1325,6 +1325,37 @@ let host_query_ha = call ~flags:[`Session]
     ~allowed_roles:_R_LOCAL_ROOT_ONLY
     ()
 
+  let notify_accept_new_pool_secret = call
```

Comment:
As discussed, let's add `hide_from_docs:true` to all of these, as they are internal calls.